### ISSUE TITLE
Issues in systemd man pages

### ISSUE THREAD

**Issue by hmartink** [systemd/systemd#17177]
Dear systemd maintainer,
the manpage-l10n project maintains a large number of translations of
man pages both from a large variety of sources (including systemd) as
well for a large variety of target languages.

During their work translators notice different possible issues in the
original (english) man pages. Sometimes this is a straightforward
typo, sometimes a hard to read sentence, sometimes this is a
convention not held up and sometimes we simply do not understand the
original.

We use several distributions as sources and update regularly (at
least every 2 month). This means we are fairly recent (some
distributions like archlinux also update frequently) but might miss
the latest upstream version once in a while, so the error might be
already fixed. We apologize and ask you to close the issue immediately
if this should be the case, but given the huge volume of projects and
the very limited number of volunteers we are not able to double check
each and every issue.

Secondly we translators see the manpages in the neutral po format,
i.e. converted and harmonized, but not the original source (be it man,
groff, xml or other). So we cannot provide a true patch (where
possible), but only an approximation which you need to convert into
your source format.

Finally the issues I'm reporting have accumulated over time and are
not always discovered by me, so sometimes my description of the
problem my be a bit limited - do not hesitate to ask so we can clarify
them.

I'm now reporting the errors for your project. If future reports
should use another channel, please let me know.
```
Man page: file-hierarchy.7
Issue: What "state" is meant here?

"During runtime, and for local configuration and state, additional "
"directories are defined:"
--
Man page: homectl.1
Issue: TZ=: correct? (Colon)

"Takes a timezone specification as string that sets the timezone for the "
"specified user\\&. Expects a `tzdata` location string\\&. When the user logs "
"in the I<$TZ> environment variable is initialized from this setting\\&. "
"Example: B<--timezone=Europe/Amsterdam> will result in the environment "
"variable \"TZ=:Europe/Amsterdam\"\\&."
--
Man page: homectl.1
Issue: "keyed by" is unclear? assigned to the key?

"Takes a path to a Linux \"hidraw\" device (e\\&.g\\&.  /dev/hidraw1), "
"referring to a FIDO2 security token implementing the \"hmac-secret\" "
"extension, that shall be able to unlock the user account\\&. If used, a "
"random salt value is generated on the host, which is passed to the FIDO2 "
"device, which calculates a HMAC hash of it, keyed by its internal secret key"
"\\&. The result is then used as key for unlocking the user account\\&. The "
"random salt is included in the user record, so that whenever authentication "
"is needed it can be passed again to the FIDO2 token, to retrieve the actual "
"key\\&."
--
Man page: homectl.1
Issue: Missing full stop at end

"Takes a password hint to store alongside the user record\\&. This string is "
"stored accessible only to privileged users and the user itself and may not "
"be queried by other users\\&. Example: B<--password-hint=\"My first pet"
"\\*(Aqs name\">"
--
Man page: homectl.1
Issue: In other files singular, i.e. Records â Record

"JSON User Records"
--
Man page: journalctl.1
Issue: contain â containing

"Removes the oldest archived journal files until the disk space they use "
"falls below the specified size (specified with the usual \"K\", \"M\", \"G\" "
"and \"T\" suffixes), or all archived journal files contain no data older "
"than the specified timespan (specified with the usual \"s\", \"m\", \"h\", "
"\"days\", \"months\", \"weeks\" and \"years\" suffixes), or no more than the "
"specified number of separate journal files remain\\&. Note that running B<--"
"vacuum-size=> has only an indirect effect on the output shown by B<--disk-"
"usage>, as the latter includes active journal files, while the vacuuming "
"operation only operates on archived journal files\\&. Similarly, B<--vacuum-"
"files=> might not actually reduce the number of journal files to below the "
"specified number, as it will not remove active journal files\\&."
--
Man page: logind.conf.5
Issue: what does Â»to kill a session scopeÂ« mean? The process in the scope?

"These settings take space-separated lists of usernames that override the "
"I<KillUserProcesses=> setting\\&. A user name may be added to "
"I<KillExcludeUsers=> to exclude the processes in the session scopes of that "
"user from being killed even if I<KillUserProcesses=yes> is set\\&. If "
"I<KillExcludeUsers=> is not set, the \"root\" user is excluded by default"
"\\&.  I<KillExcludeUsers=> may be set to an empty value to override this "
"default\\&. If a user is not excluded, I<KillOnlyUsers=> is checked next\\&. "
"If this setting is specified, only the session scopes of those users will be "
"killed\\&. Otherwise, users are subject to the I<KillUserProcesses=yes> "
"setting\\&."
--
Man page: networkctl.1
Issue: In the links document nothing is discussed at all. It rather notes taht the standard is worth debating and request for improvement are sought.

"Show numerical address labels that can be used for address selection\\&. "
"This is the same information that B<ip-addrlabel>(8)  shows\\&. See "
"\\m[blue]B<RFC 3484>\\m[]\\&\\s-2\\u[1]\\d\\s+2 for a discussion of address "
"labels\\&."
--
Man page: pam_systemd.8
Issue: concurrent implies that there are more than one session which seems odd (here and the following strings), concurrent â current?

"A new systemd scope unit is created for the session\\&. If this is the first "
"concurrent session of the user, an implicit per-user slice unit below user"
"\\&.slice is automatically created and the scope placed into it\\&. An "
"instance of the system service user@\\&.service, which runs the systemd user "
"manager instance, is started\\&."
--
Man page: repart.d.5
Issue: B<systemd-makefs> â B<systemd-makefs>(8)

"Note that these partition definition files do not describe the contents of "
"the partitions, such as the file system used\\&. Separate mechanisms, such "
"as B<systemd-growfs>(8)  and B<systemd-makefs> maybe be used to initialize "
"or grow the file systems inside of these partitions\\&."
--
Man page: resolvectl.1
Issue: of â or

"Specifies the network interface to execute the query on\\&. This may either "
"be specified as numeric interface index or as network interface string (e\\&."
"g\\&.  \"en0\")\\&. Note that this option has no effect if system-wide DNS "
"configuration (as configured in /etc/resolv\\&.conf or /etc/systemd/resolve"
"\\&.conf) in place of per-link configuration is used\\&."
--
Man page: resolved.conf.5
Issue: Site-private == private? https://docs.microsoft.com/de-de/azure/dns/private-dns-overview

"Site-private DNS zones generally conflict with DNSSEC operation, unless a "
"negative (if the private zone is not signed) or positive (if the private "
"zone is signed) trust anchor is configured for them\\&. If \"allow-downgrade"
"\" mode is selected, it is attempted to detect site-private DNS zones using "
"top-level domains (TLDs) that are not known by the DNS root server\\&. This "
"logic does not work in all private zone setups\\&."
--
Man page: sysctl.d.5
Issue: The first sentence contradicts the third sentence

"Any access permission errors and attempts to write variables not present on "
"the local system are logged at debug level and do not cause the service to "
"fail\\&. Moreover, if a variable assignment is prefixed with a single \"-\" "
"character, failure to set the variable for other reasons will be logged at "
"debug level and will not cause the service to fail\\&. In other cases, "
"errors when setting variables are logged with higher priority and cause the "
"service to return failure at the end (after processing other variables)\\&."
--
Man page: systemctl.1
Issue: No section Environment?

"Depending on whether B<--system> (the default), B<--user>, or B<--global> is "
"specified, this command creates a drop-in file for each unit either for the "
"system, for the calling user, or for all futures logins of all users\\&. "
"Then, the editor (see the \"Environment\" section below) is invoked on "
"temporary files which will be written to the real location if the editor "
"exits successfully\\&."
--
Man page: systemd.1
Issue: of of â of

"These variables may contain a list of paths, separated by colons (\":\")\\&. "
"When set, if the list ends with an empty component (\"\\&.\\&.\\&.:\"), this "
"list is prepended to the usual set of of paths\\&. Otherwise, the specified "
"list replaces the usual set of paths\\&."
--
Man page: systemd.1
Issue: Superfluos comma before "environment"

"Controls log output, with the same effect as the I<$SYSTEMD_LOG_COLOR>, I<"
"$SYSTEMD_LOG_LEVEL>, I<$SYSTEMD_LOG_LOCATION>, I<$SYSTEMD_LOG_TARGET>, I<"
"$SYSTEMD_LOG_TIME>, environment variables described above\\&.  I<systemd\\&."
"log_color>, I<systemd\\&.log_location>, and I<systemd\\&.log_time> can be "
"specified without an argument, with the same effect as a positive boolean\\&."
--
Man page: systemd.1
Issue: There is no section "Environment" in this man page? "Known Environment Variables"?

"When B<systemd> is used as a user manager, the kernel command line is "
"ignored and the options described are understood\\&. Nevertheless, "
"B<systemd> is usually started in this mode through the B<user@.service>(5)  "
"service, which is shared between all users, and it may be more convenient to "
"use configuration files to modify settings, see B<systemd-user.conf>(5), or "
"a drop-in that specifies one of the environment variables listed above in "
"the Environment section, see B<systemd.unit>(5)\\&."
--
Man page: systemd.directives.7
Issue: Why are these two quoted (but not others)?

"B<\\*(Aqh\\*(Aq>"

"B<\\*(Aqs\\*(Aq>"

"B<\\*(Aqy\\*(Aq>"
--
Man page: systemd.directives.7
Issue: Double slash?

"/foo//bar/baz/"
--
Man page: systemd.exec.5
Issue 1: encoding â¦ DER encoded signature?
Issue 2: signature of the root hash signature â signature of the root hash

"Takes a PKCS7 formatted binary signature of the I<RootHash=> option as a "
"path to a DER encoded signature file or as an ASCII base64 string encoding "
"of the DER encoded signature, prefixed by \"base64:\"\\&. The dm-verity "
"volume will only be opened if the signature of the root hash signature is "
"valid and created by a public key present in the kernel keyring\\&. If this "
"option is not specified, but a file with the \\&.roothash\\&.p7s suffix is "
"found next to the image file, bearing otherwise the same name (except if the "
"image has the \\&.raw suffix, in which case the signature file must not have "
"it in its name), the signature is read from it and automatically used\\&."
--
Man page: systemd.exec.5
Issue: This setting not â This setting does not

"Takes a profile name as argument\\&. The process executed by the unit will "
"switch to this profile when started\\&. Profiles must already be loaded in "
"the kernel, or the unit will fail\\&. If prefixed by \"-\", all errors will "
"be ignored\\&. This setting has no effect if AppArmor is not enabled\\&. "
"This setting not affect commands prefixed with \"+\"\\&."
--
Man page: systemd.exec.5
Issue: fork -> B<fork>(2)

"Takes a boolean argument\\&. If true, elevated CPU scheduling priorities and "
"policies will be reset when the executed processes fork, and can hence not "
"leak into child processes\\&. See B<sched_setscheduler>(2)  for details\\&. "
"Defaults to false\\&."
--
Man page: systemd.exec.5
Issue: int â in

"These options take a whitespace-separated list of directory names\\&. The "
"specified directory names must be relative, and may not include \"\\&.\\&."
"\"\\&. If set, one or more directories by the specified names will be "
"created (including their parents) below the locations defined in the "
"following table, when the unit is started\\&. Also, the corresponding "
"environment variable is defined with the full path of directories\\&. If "
"multiple directories are set, then in the environment variable the paths are "
"concatenated with colon (\":\")\\&."
--
Man page: systemd.exec.5
Issue: it that is â it that are

"Takes a boolean argument\\&. If true, sets up a new file system namespace "
"for the executed processes and mounts private /tmp/ and /var/tmp/ "
"directories inside it that are not shared by processes outside of the "
"namespace\\&. This is useful to secure access to temporary files of the "
"process, but makes sharing between processes via /tmp or /var/tmp impossible"
"\\&. If this is enabled, all temporary files created by a service in these "
"directories will be removed after the service is stopped\\&. Defaults to "
"false\\&. It is possible to run two or more units within the same private /"
"tmp and /var/tmp namespace by using the I<JoinsNamespaceOf=> directive, see "
"B<systemd.unit>(5)  for details\\&. This setting is implied if "
"I<DynamicUser=> is set\\&. For this setting the same restrictions regarding "
"mount propagation and privileges apply as for I<ReadOnlyPaths=> and related "
"calls, see above\\&. Enabling this setting has the side effect of adding "
"I<Requires=> and I<After=> dependencies on all mount units necessary to "
"access /tmp and /var/tmp\\&. Moreover an implicitly I<After=> ordering on "
"B<systemd-tmpfiles-setup.service>(8)  is added\\&."
--
Man page: systemd.exec.5
Issue: used, contains â used, it contains

"If the I<LogNamespace=> service setting is used, contains name of the "
"selected logging namespace\\&."
--
Man page: systemd-firstboot.1
Issue: First sentence should also mention shell now?

"Query the user for locale, keymap, timezone, hostname and root password\\&. "
"This is equivalent to specifying B<--prompt-locale>, B<--prompt-keymap>, B<--"
"prompt-timezone>, B<--prompt-hostname>, B<--prompt-root-password>, B<--"
"prompt-root-shell> in combination\\&."

"Copy locale, keymap, time zone and root password from the host\\&. This is "
"equivalent to specifying B<--copy-locale>, B<--copy-keymap>, B<--copy-"
"timezone>, B<--copy-root-password>, B<--copy-root-shell> in combination\\&."
--
Man page: systemd-id128.1
Issue: verb â word

"B<id128> may be used to conveniently print B<sd-id128>(3)  UUIDs\\&. What "
"identifier is printed depends on the specific verb\\&."
--
Man page: systemd.journal-fields.7
Issue: the the â the

"The control group path in the systemd hierarchy, the the systemd slice unit "
"name, the systemd unit name, the unit name in the systemd user manager (if "
"any), the systemd session ID (if any), and the owner UID of the systemd user "
"unit or systemd session (if any) of the process the journal entry originates "
"from\\&."
--
Man page: systemd.journal-fields.7
Issue: verb missing in second sentence, sencond half

"The kernel device name\\&. If the entry is associated to a block device, the "
"major and minor of the device node, separated by \":\" and prefixed by \"b"
"\"\\&. Similar for character devices but prefixed by \"c\"\\&. For network "
"devices, this is the interface index prefixed by \"n\"\\&. For all other "
"devices, this is the subsystem name prefixed by \"+\", followed by \":\", "
"followed by the kernel device name\\&."
--
Man page: systemd-machined.service.8
Issue: Misplace full stop near the end (and missing at real end)

"The daemon provides both a C library interface (which is shared with "
"B<systemd-logind.service>(8)) as well as a D-Bus interface\\&. The library "
"interface may be used to introspect and watch the state of virtual machines/"
"containers\\&. The bus interface provides the same but in addition may also "
"be used to register or terminate machines\\&. For more information please "
"consult B<sd-login>(3)  and B<org.freedesktop.machine1>(5)\\&. and B<org."
"freedesktop.LogControl1>(5)"
--
Man page: systemd-mount.1
Issue: file system is â file system type is

"Specifies the file system type to mount (e\\&.g\\&.  \"vfat\", \"ext4\", "
"\\&...)\\&. If omitted (or set to \"auto\") the file system is determined "
"automatically\\&."
--
Man page: systemd.netdev.5
Issue: No such section ??

"The netdev kind\\&. This setting is compulsory\\&. See the \"Supported "
"netdev kinds\" section for the valid keys\\&."
--
Man page: systemd.netdev.5
Issue: bases â¦ based on? Is it port â ports?

"Configures VXLAN port range\\&. VXLAN bases source UDP port based on flow to "
"help the receiver to be able to load balance based on outer header flow\\&. "
"It restricts the port range to the normal UDP local ports, and allows "
"overriding via configuration\\&."
--
Man page: systemd.netdev.5
Issue: in â as in
 
"Accepts the same key in [VXLAN] section\\&."
--
Man page: systemd.netdev.5
Issue: Second sentence correct? (Shouldn't it reference TunnelId= instead of PeerTunnelId=?)
 
"Specifies the peer tunnel id\\&. Takes a number in the range "
"1\\(em4294967295\\&. The value used must match the \"PeerTunnelId=\" value "
"being used at the peer\\&. This setting is compulsory\\&."
--
Man page: systemd.netdev.5
Issue: in â as in

"Accepts the same key in [MACsecReceiveChannel] section\\&."

"Accepts the same key in [MACsecTransmitAssociation] section\\&."
--
Man page: systemd.netdev.5
Issue: will be removed â removed

"Specifies the port number, where the IP encapsulation packets will arrive"
"\\&. Please take note that the packets will arrive with the encapsulation "
"will be removed\\&. Then they will be manually fed back into the network "
"stack, and sent ahead for delivery to the real destination\\&. This option "
"is mandatory\\&."
--
Man page: systemd.net-naming-scheme.7
Issue: Markup incorrect (here and in the following) I<ID_NET_NAME_ONBOARD=> â B<ID_NET_NAME_ONBOARD=>

"I<ID_NET_NAME_ONBOARD=>I<prefix>B<o>I<number>"
--
Man page: systemd.network.5
Issue: "more than once", later "multiple times"

"A DNS server address, which must be in the format described in "
"B<inet_pton>(3)\\&. This option may be specified more than once\\&. Each "
"address can optionally take a port number separated with \":\", a network "
"interface name or index separated with \"%\", and a Server Name Indication "
"(SNI) separated with \"#\"\\&. When IPv6 address is specified with a port "
"number, then the address must be in the square brackets\\&. That is, the "
"acceptable full formats are \"111\\&.222\\&.333\\&.444:9953%ifname#example"
"\\&.com\" for IPv4 and \"[1111:2222::3333]:9953%ifname#example\\&.com\" for "
"IPv6\\&. This setting can be specified multiple times\\&. If an empty string "
"is assigned, then the all previous assignments are cleared\\&. This setting "
"is read by B<systemd-resolved.service>(8)\\&."
--
Man page: systemd.network.5
Issue: and will â which will

"When configured, allows to set arbitrary request options in the DHCPv6 "
"request options list and will sent to the DHCPV6 server\\&. A whitespace-"
"separated list of integers in the range 1\\&.\\&.254\\&. Defaults to unset"
"\\&."
--
Man page: systemd.network.5
Issue: take â takes

"When true (the default), the onlink prefix received in the Router "
"Advertisement will be used and take precedence over any statically "
"configured ones\\&."
--
Man page: systemd.network.5
Issue: \\\\&\. â :

"The [LLDP] section manages the Link Layer Discovery Protocol (LLDP) and "
"accepts the following keys\\&."
--
Man page: systemd.network.5
Issue: Device â device

"Controls support for Ethernet LLDP packet\\*(Aqs Manufacturer Usage "
"Description (MUD)\\&. MUD is an embedded software standard defined by the "
"IETF that allows IoT Device makers to advertise device specifications, "
"including the intended communication patterns for their device when it "
"connects to the network\\&. The network can then use this intent to author a "
"context-specific access policy, so the device functions only within those "
"parameters\\&. Takes an URL of length up to 255 characters\\&. A superficial "
"verification that the string is a valid URL will be performed\\&. See "
"\\m[blue]B<RFC 8520>\\m[]\\&\\s-2\\u[17]\\d\\s+2 for details\\&. The MUD URL "
"received from the LLDP packets will be saved at the state files and can be "
"read via B<sd_lldp_neighbor_get_mud_url()> function\\&."
--
Man page: systemd.network.5
Issue: to prevent -> prevents

"Specifies the hard limit on the FIFO size in bytes\\&. The size limit (a "
"buffer size) to prevent it from overflowing in case it is unable to dequeue "
"packets as quickly as it receives them\\&. When this limit is reached, "
"incoming packets are dropped\\&. When suffixed with K, M, or G, the "
"specified size is parsed as Kilobytes, Megabytes, or Gigabytes, "
"respectively, to the base of 1024\\&. Defaults to unset and kernel\\*(Aqs "
"default is used\\&."

"Specifies the hard limit on the FIFO size in number of packets\\&. The size "
"limit (a buffer size) to prevent it from overflowing in case it is unable to "
"dequeue packets as quickly as it receives them\\&. When this limit is "
"reached, incoming packets are dropped\\&. An unsigned integer in the range "
"0\\(en4294967294\\&. Defaults to unset and kernel\\*(Aqs default is used\\&."
--
Man page: systemd.network.5
Issue: must be 0..255 â must be in the range 0..255

"The priority map maps the priority of a packet to a band\\&. The argument is "
"a white-space separated list of numbers\\&. The first number indicates which "
"band the packets with priority 0 should be put to, the second is for "
"priority 1, and so on\\&. There can be up to 16 numbers in the list\\&. If "
"there are fewer, the default band that traffic with one of the unmentioned "
"priorities goes to is the last one\\&. Each band number must be 0\\&."
"\\&.255\\&. This setting can be specified multiple times\\&. If an empty "
"string is assigned, then the all previous assignments are cleared\\&."
--
Man page: systemd.network.5
Issue: than one â than once

"A whitespace-separated list of hardware address of the currently connected "
"wireless LAN\\&. Use full colon-, hyphen- or dot-delimited hexadecimal\\&. "
"See the example in I<MACAddress=>\\&. This option may appear more than one, "
"in which case the lists are merged\\&. If the empty string is assigned to "
"this option, the list of BSSID defined prior to this is reset\\&."
--
Man page: systemd-notify.1
Issue: B<systemd-notify > â B<systemd-notify>

"Do not synchronously wait for the requested operation to finish\\&. Use of "
"this option is only recommended when B<systemd-notify> is spawned by the "
"service manager, or when the invoking process is directly spawned by the "
"service manager and has enough privileges to allow B<systemd-notify > to "
"send the notification on its behalf\\&. Sending notifications with this "
"option set is prone to race conditions in all other cases\\&."
--
Man page: systemd-nspawn.1
Issue: encoding the DER encoded signatur? One encoded too much?

"Takes a PKCS7 formatted binary signature of the B<--root-hash=> option as a "
"path to a DER encoded signature file or as an ASCII base64 string encoding "
"of the DER encoded signature, prefixed by \"base64:\"\\&. The dm-verity "
"volume will only be opened if the signature of the root hash hex string is "
"valid and done by a public key present in the kernel keyring\\&. If this "
"option is not specified, but a file with the \\&.roothash\\&.p7s suffix is "
"found next to the image file, bearing otherwise the same name (except if the "
"image has the \\&.raw suffix, in which case the signature file must not have "
"it in its name), the signature is read from it and automatically used\\&."
--
Man page: systemd.service.5
Issue: B markup missing for last words

"B<Table\\ \\&2.\\ \\&Exit causes and the effect of the >I<Restart=> settings "
"on them"
--
Man page: systemd.socket.5
Issue: Is the reference ok?

"Socket units may be used to implement on-demand starting of services, as "
"well as parallelized starting of services\\&. See the blog stories linked at "
"the end for an introduction\\&."
--
Man page: systemd.socket.5
Issue: functionfs â FunctionFS

"Specifies a \\m[blue]B<USB FunctionFS>\\m[]\\&\\s-2\\u[1]\\d\\s+2 endpoints "
"location to listen on, for implementation of USB gadget functions\\&. This "
"expects an absolute file system path of functionfs mount point as the "
"argument\\&. Behavior otherwise is very similar to the I<ListenFIFO=> "
"directive above\\&. Use this to open the FunctionFS endpoint ep0\\&. When "
"using this option, the activated service has to have the "
"I<USBFunctionDescriptors=> and I<USBFunctionStrings=> options set\\&."
--
Man page: systemd.special.7
Issue: Superfluous ". RE" at the end

"The XDG specification defines a way to autostart applications using XDG "
"desktop files\\&. systemd ships B<systemd-xdg-autostart-generator>(8)  for "
"the XDG desktop files in autostart directories\\&. Desktop Environments can "
"opt-in to use this service by adding a I<Wants=> dependency on \"xdg-desktop-"
"autostart\\&.target\" \\&.  .RE"
--
Man page: timesyncd.conf.5
Issue: To the translators the term "root distance" is unclear

"Maximum acceptable root distance\\&. Takes a time value (in seconds)\\&. "
"Defaults to 5 seconds\\&."
```

**Comment by poettering** [comment]
Any chance you can submit this as github PR? i.e. as proper patch? This is usually how we receive contributions both in docs and code!

**Comment by keszybz** [comment]
I pushed #17195 which fixes a majority of the issues listed here.
In a few cases problems which have already been fixed were reported.

I didn't touch the following:
```
Man page: systemd.netdev.5
Issue: bases â¦ based on? Is it port â ports?

"Configures VXLAN port range\\&. VXLAN bases source UDP port based on flow to "
"help the receiver to be able to load balance based on outer header flow\\&. "
"It restricts the port range to the normal UDP local ports, and allows "
"overriding via configuration\\&."

Man page: homectl.1
Issue: "keyed by" is unclear? assigned to the key?

"Takes a path to a Linux \"hidraw\" device (e\\&.g\\&.  /dev/hidraw1), "
"referring to a FIDO2 security token implementing the \"hmac-secret\" "
"extension, that shall be able to unlock the user account\\&. If used, a "
"random salt value is generated on the host, which is passed to the FIDO2 "
"device, which calculates a HMAC hash of it, keyed by its internal secret key"

Man page: logind.conf.5
Issue: what does Â»to kill a session scopeÂ« mean? The process in the scope?
```

Those are correct already:
```
Man page: journalctl.1
Issue: contain â containing
Man page: systemd.socket.5
Issue: Is the reference ok?
```
Also there were a few cases where a section was reported to be missing, but in fact it appear in the page.

**Comment by hmartink** [comment]
Thanks for fixing those issues.

I tried to avoid reporting fixed issues again, but given the large volume of man pages both within systemd and the various distributions we support (which are not always up to date) I rereported some again, I apologize.

In the same vain: We have 110 (yes, one hundert and ten) man pages sources (i.e. projects). Systemd is definitely one of the largest ones, but each project is different and has a different structure (and there are several types of source format). And I'm currently the only one trying to get  those fixes upstream again. So I'm unable to translate the neutral texts back to each projects native format.

Also some fixes are straightforward, but many actually require a deeper insight (i.e. I cannot simply provide updated text). Therefor I'm unable to provide some kind of "diff" or merge request, I'm sorry. 

Since the work flow we have is completely text based, the ideal interface for me would be some kind of e-mail address (list, individual, to an issue tracker etc.), because this would cause me the least pain. But I'm fine with pasting the text output in this issue tracker, if this is the best compromise between our two projects.

Thanks again for your swift handling of the issues.

Helge
